data/data/openstack/variables-openstack.tf

```diff
@@ -1,3 +1,15 @@
+variable "openstack_master_root_volume_type" {
+  type        = string
+  default     = null
```

Comment:
i don't think `null` is required as default here. see https://www.terraform.io/docs/configuration/expressions.html#null

Reply:
I think its easier to work with as null, but what would you suggest?

Reply:
won't null be the implicit default, without needing this line to make it the explicit default?

Reply:
It was throwing an error without the default

Reply:
do you still have the error handy?  Nice to say "you might not think you need the null defaults, but without them it fails with ..." in commit messages and things

Reply:
Error: Unassigned variable: The input variable "openstack_master_root_volume_size" has not been assigned a value. This is a bug in Terraform; please report it in a GitHub issue.

Reply:
I tend to shoot for flat commits unless I add something fundamentally different, I assumed that was what you guys wanted

Reply:
> I tend to shoot for flat commits...

What's a flat commit?  [Our contributing docs][1] ask for what and why, so if anything is finicky or there was discussion around it, it's nice (but not strictly required) to summarize in the commit message to help pass the motivation on to future devs.  So single commit that includes:

> Without the null defaults, install errors with:
>
>     Error: Unassigned variable: The input variable "openstack_master_root_volume_size" has not been assigned a value. This is a bug in Terraform; please report it in a GitHub issue.

saves folks blaming the line the trouble of guessing whether they are overlooking something or the original author overlooked it ;).

[1]: https://github.com/openshift/installer/blob/master/CONTRIBUTING.md#commit-message-format